website/markdown/docs/commands/signing.mdx

```diff
@@ -18,12 +18,13 @@ Tuist aims to solve all of the above pain points in a simple and _deterministic_
 
 ## How to Get Started
 
-As the first step, download all your provisioning profiles and certficates that you'll need.
-Once you download them, make sure to rename the files, so they abide by the following convention:
-`Target.Configuration.extension`
-Where `Target` should be a name of the target, `Configuration` a name of the configuration and leave the default extension
-that you downloaded the file with (eg. `p12` for certificate, etc.).
-Now you can put all those files in `Tuist/Signing` directory.
+As the first step, download all your provisioning profiles that you'll need.
+Once you downloaded them, make sure to rename the provisioning profiles, so they abide by the following convention:
+`Target.Configuration.mobileprovision`
```

Comment:
`mobileprovision` is not strictly right as we support `macOS`, too, for which the extension is `provisionprofile`